_includes/_guides/_solo_crunching.htm

```diff
@@ -0,0 +1,263 @@
+                  </li>
+              </ul>
+          </div><!-- .step-body -->
+      </section><!-- .step -->
+
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-sync-blockchain.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">05</span>
+                  <strong>Step 5.</strong> Sync with the Blockchain
+              </h3>
+          </header>
+          <div class="step-body">
+              <p>
+                  Before your wallet can do anything, it needs to be synced
+                  (the green check mark under connections means you are synced).
+              </p>
+          </div><!-- .step-body -->
+      </section><!-- .step -->
+
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-get-gridcoin.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">06</span>
+                  <strong>Step 6.</strong> Get Some Gridcoin
+              </h3>
+          </header>
+          <div class="step-body">
+              <p>
+                  In Gridcoin's Proof-of-Stake (PoS) system, BOINC rewards are
+                  given when users stake along with an additional 10 GRC.
+              </p>
+              <p>
+                  To stake and earn your BOINC rewards, a balance of GRC is
+                  required.
+              </p>
+
+              <ol>
+                  <li>
+                      For monthly staking, a balance of approximately 5000 GRC
+                      is recommended.
+                  </li>
+                  <li>
+                      The rate you are rewarded for your BOINC work is the rate
+                      you mint PoS blocks because BOINC rewards are layered on
+                      top of these blocks.
+                  </li>
+                  <li>
+                      Your balance does not affect the amount of the payout,
+                      only how frequent. You need to wait around 16 hours for
+                      received coins to become eligible to earn staking rewards.
+                  </li>
+              </ol>
+
+              <p>
+                  If you don't want to buy any Gridcoin, you can get some through
+                  the faucets or through <a href="#">pool mining</a>.
+                  On most exchanges, you need to buy Bitcoin first in order to
+                  buy Gridcoin.
+              </p>
+          </div><!-- .step-body -->
+      </section><!-- .step -->
+
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-earn-grc.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">07</span>
+                  <strong>Step 7.</strong> Earn Research Rewards
+              </h3>
+          </header>
+          <div class="step-body">
+              <h4 class="h5">Check Your Projects</h4>
```

Comment:
Might want to mention the GDPR requirements some projects have somewhere in here

Reply:
Hm, good point. Also I think the GDPR should be mentioned on whitelist page also. Perhaps it worth to create an to extend the whitelist page.

Reply:
I created the issue with a requirements to extend whitelist page with GDPR requirements.
Updates at this section is a part of that task.